src/CheckFor.php

```diff
@@ -210,4 +210,98 @@ public static function isType($desired_type, $row) {
     }
     return TRUE;
   }
+
+  /**
+   * Check file for redirects.
+   *
+   * @param object $row
+   *   A row object as delivered by migrate.
+   *
+   * @return bool
+   *   TRUE - full URI(s) of the redirect destination(s).
```

Comment:
This should only return one URL.  A page can't have two outgoing redirects.  Should probably say:
`string - full URL of the redirect destination.`